### PR TITLE
Verkoper dashboard

### PR DESCRIPTION
**Pull request by ValenRemZP**
Issue #60 

*Added route to dashboard in routes.php.
*Added dashboard shortcut in nav.php.
*Added verkoperDashboard.php.

-Available in dashboard: 
| List of products from the logged in seller with options to delete, add or edit the product.
| Template for the review system yet to develop to show the logged in use (seller/auctioneer) his personal rating and reviews he's gotten from their buyer/wining bidders.

### CODE REVIEW

**Review by CedricVerlinden — COMMENTED** [review]
LGTM! Will make some clean up changes!

File: src/public/member/verkoperDashboard.php [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+      margin-bottom: 0.6rem;
+    }
+
+    .top {
+      align-items: center;
+      flex-direction: column;
+      justify-content: center;
+    }
+  }
+</style>
```
Comment (CedricVerlinden) [inline src/public/member/verkoperDashboard.php:147]: Replace all these css styles with tailwind classes (https://tailwindcss.com/docs/padding), just look up the css properties used in the file, on the tailwind website and use those as classes for the corresponding html tags.

I've written comments behind the css properties of the box class as an example, those classes also have to be added to all divs/elements that have the 'box' class

**Review by CedricVerlinden — CHANGES_REQUESTED** [review]

File: src/public/member/verkoperDashboard.php (hunk 1 above)
Comment (ValenRemZP) [reply src/public/member/verkoperDashboard.php:147]: I've tried changing the css properties to tailwind classes, but it changes from drastically to barely noticeable parts of the design.

Comment (CedricVerlinden) [reply src/public/member/verkoperDashboard.php:147]: mhh alright, I'll take a look.

**Comment by CedricVerlinden** [comment]
Closed due to #72

**Pull request closed without merging**